price-feeder/oracle/provider/binance.go

```diff
@@ -191,13 +221,27 @@ func (p *BinanceProvider) setTickerPair(ticker BinanceTicker) {
 func (p *BinanceProvider) setCandlePair(candle BinanceCandle) {
 	p.mtx.Lock()
 	defer p.mtx.Unlock()
-	p.candles[candle.Symbol] = candle
+	timePeriod := PastUnixTime(providerCandlePeriod)
```

Comment:
nit (naming): is `timePeriod` basically functioning as something like `staleTime`?

Reply:
Changed those to `staleTime` 👍